app/src/components/page/Sidebar/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { MenuItems } from '.';
+
+export const getActiveKey = (menus: MenuItems[], loc: string[]) => {
```

Comment:
it would be nice to see some docstrings for this, and a unit tests, the menus, above can be moved to their own file, to facilitate testing.

Reply:
@p-netm might be abit difficult to move the schema to utils because of [this](https://github.com/OpenSRP/web/pull/395/files#diff-552ba8f96c6d579aa4f53424941f377eeb60e0f25fca23ed7ce15a10ffb330dcR147)

Reply:
added tests for the util functon here https://github.com/OpenSRP/web/pull/395/files#diff-e4f8b8625255d908bdd20b793fc48fdbcbd39ad0ed8af544fded45e2f9c76273R158